app/buyers/views/buyers.py

```diff
@@ -270,59 +279,229 @@ def view_brief_responses(framework_slug, lot_slug, brief_id):
+                else:
+                    sheet.get_row(question.id).write_cell(response[question.id])
 
-    sorted_brief_responses = get_sorted_responses_for_brief(brief, data_api_client)
+        return doc
 
-    content = content_loader.get_manifest(brief['frameworkSlug'], 'output_brief_response').filter(
-        {'lot': brief['lotSlug']}
-    )
-    section = content.get_section('view-response-to-requirements')
-
-    column_headings = []
-    question_key_sequence = []
-    boolean_list_questions = []
-    csv_rows = []
-
-    # Build header row from manifest and add it to the list of rows
-    for question in section.questions:
-        question_key_sequence.append(question.id)
-        if question['type'] == 'boolean_list' and brief.get(question.id):
-            column_headings.extend(brief[question.id])
-            boolean_list_questions.append(question.id)
-        else:
-            column_headings.append(question.name)
-    csv_rows.append(column_headings)
-
-    # Add a row for each eligible response received
-    for brief_response in sorted_brief_responses:
-        if all_essentials_are_true(brief_response):
-            row = []
-            for key in question_key_sequence:
-                if key in boolean_list_questions:
-                    row.extend(brief_response.get(key))
-                else:
-                    row.append(brief_response.get(key))
-            csv_rows.append(row)
-
-    return Response(
-        csv_generator.iter_csv(csv_rows),
-        mimetype='text/csv',
-        headers={
-            "Content-Disposition": "attachment;filename=responses-to-requirements-{}.csv".format(brief['id']),
-            "Content-Type": "text/csv; header=present"
-        }
-    ), 200
+    def create_ods_response(self, context=None):
+        buf = BytesIO()
+
+        self.generate_ods(context['brief'], context['responses']).save(buf)
+
+        return Response(
+            buf.getvalue(),
+            mimetype='application/vnd.oasis.opendocument.spreadsheet',
+            headers={
+                "Content-Disposition": (
+                    "attachment;filename=responses-to-requirements-{}.ods"
+                ).format(context['brief']['id']),
+                "Content-Type": "application/vnd.oasis.opendocument.spreadsheet"
+            }
+        ), 200
+
+    def create_response(self, context=None):
+        responses = context['responses']
+
+        if responses and 'essentialRequirementsMet' in responses[0]:
+            return self.create_ods_response(context)
+
+        return self.create_csv_response(context)
+
+    def dispatch_request(self, **kwargs):
+        context = self.get_context_data(**kwargs)
+
+        return self.create_response(context)
+
+
+buyers.add_url_rule('/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/responses/download',
+                    view_func=DownloadView.as_view(str('download_brief_responses')),
+                    methods=['GET'])
```

Comment:
because unicode_litterals is stupid and breaks flask in places, this is one...

that value gets assigned as the name of the function so routing treats it the same `def download_brief_responses` but you cant have unicode function names in python2